vars/deployApp.groovy

```diff
@@ -31,6 +32,9 @@ def call(Map parameters = [:]) {
     if (!env) {
         error 'you must provide a valid environment name'
     }
+    if (!region) {
+        error 'you must provide a supported region name'
+    }
```

Comment:
Added conditional statement to check `region` is not null.

Reply:
For changes that need to be coordinated across multiple projects I would generally recommend looking for ways to implement the change incrementally. That way other developers won't get disrupted if an unexpected problem occurs part-way through. This is a pretty simple change so the risk is quite low, but I think this is a good habit to get into. A possible approach:

1. Add optional `region` parameter which defaults to `us-west-1` if missing
2. Change `deployApp` calls in other projects to explicitly specify the region and verify that deployment of each project works after the change
3. Once all `deployApp` calls are migrated, make the `region` parameter required

We could also stop after step 1 and always assume us-west-1 as a default, since we expect it to be the main region for the foreseeable future. I can see an argument for making it explicit whenever `deployApp` is called though.

Reply:
The conditional step has been removed. We no longer check for the existence of `region`. I have added another comment below, but we are now defaulting the value of `region` to be `us-west-1` in the scenario where `parameters.region` has not been provided.